src/errors/test-run/templates.js

```diff
@@ -78,12 +78,31 @@ export default {
         ${err.getCallsiteMarkup()}
     `),
 
+    [TYPE.selectActionSelectorTypeError]: err => markup(err, `
+        The ${err.argumentName} argument is expected to be a string, but it was <code>${err.actualType}</code>.
+
+        ${err.getCallsiteMarkup()}
+    `),
+
     [TYPE.actionStringArgumentError]: err => markup(err, `
-        The <code>${err.argumentName}</code> argument is expected to be a non-empty string, but it was ${err.actualType !== 'empty' ? `<code>${err.actualType}</code>` : err.actualType}.
+        The <code>${err.argumentName}</code> argument is expected to be a non-empty string, but it was ${err.actualValue !==
```

Comment:
Why we need `selectActionSelectorTypeError` when we have this error? Use `""` instead of `empty`